packages/seedless-onboarding-controller/src/SeedlessOnboardingController.ts

```diff
@@ -235,29 +250,34 @@ export class SeedlessOnboardingController<EncryptionKey> extends BaseController<
     // assert that the user is authenticated before creating the TOPRF key and backing up the seed phrase
     this.#assertIsAuthenticatedUser(this.state);
 
-    // locally evaluate the encryption key from the password
-    const { encKey, authKeyPair, oprfKey } = this.toprfClient.createLocalKey({
-      password,
-    });
+    return this.#withControllerLock(async () => {
```

Comment:
Also at every other place using `return this.withControllerLock`, pls replace with `return await this.withControllerLock` to align with mm code guidelines